### PR TITLE
Feature/refactor post fetch

### PR DESCRIPTION
**Pull request by piperjarrett**
What does this do?
Adds DOM manipulation when a post is made. Updates the new data without refreshing the page.
Where to start? 
apiCalls.js and scripts.js
How was it tested?
By the DOM updating
Additional notes: 
I also removed the <p> from when I was going to use them in error handling.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ let sleepData;
 let activityData;
 let dateInput;
 let myChart = null;
+let result;
```
Comment (Jedeo) [inline src/scripts.js:34]: Thank you for doing this

File: src/scripts.js [hunk 2]
```diff
@@ -178,32 +190,41 @@ function getDataToPost(event) {
...
       date: calenderDate,
       numSteps: parseInt(minsActive.value),
       minutesActive: parseInt(numOfSteps.value),
       flightsOfStairs: parseInt(flightsOfStairs.value),
     };
-    postData("activity", data);
+    detail = "activity";
   }
-  promiseAll();
+  console.log(detail);
```
Comment (Jedeo) [inline src/scripts.js]: looks like we got some console logs here

File: src/scripts.js [hunk 3]
```diff
@@ -264,22 +285,22 @@ function displayFriends() {
...
-  )} hours </p>`;
+  ${user.userSleepData.getAvgSleepDataPerDay("hoursSlept")} hours </p>`;
   avgQualitySleep.innerHTML = `<p>Average Sleep Quality:<br>${user.userSleepData.getAvgSleepDataPerDay(
     "sleepQuality"
   )}</p>`;
 }
 
 function displaySleepForAWeek() {
   formatInputDate();
+  console.log(dateInput);
```
Comment (Jedeo) [inline src/scripts.js]: console log

**Review by Jedeo — APPROVED** [review]
I don't see anything I would  add. for the console logs you can just remove them on GitHub. thank you again for the help

File: src/scripts.js (hunk 3 above)
Comment (piperjarrett) [reply src/scripts.js]: done!

File: src/scripts.js (hunk 2 above)
Comment (piperjarrett) [reply src/scripts.js]: done!

File: src/scripts.js [hunk 4]
```diff
@@ -124,11 +124,24 @@ changeBackground.addEventListener("click", () => {
 });
 
 promiseAll().then((responses) => {
+  assignData(responses);
+  createUser();
+  createClasses(user);
```
Comment (forsethnico) [inline src/scripts.js]: Here you are passing in user as argument but in line 140 createUser function you are not, is this a typo or on purpose?

Comment (piperjarrett) [reply src/scripts.js]: done!

**Pull request merged**